app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    @abstractmethod
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
```

Comment:
Are you sure that `__init__` is abstractmethod?)
The same in other similar cases